src/main/java/com/jelly/farmhelperv2/feature/impl/AutoSprayonator.java

```diff
@@ -193,7 +193,11 @@ public void onTickMainState(TickEvent.ClientTickEvent event) {
     @SubscribeEvent
     public void onChatReceive(ClientChatReceivedEvent event) {
         if (event.type != 0 || !this.enabled) return;
-        String message = StringUtils.stripControlCodes(event.message.getUnformattedText());
+        String chatMessage = event.message.getUnformattedTextForChat();
```

Comment:
Same as above

Reply:
ah crap i never had to use this sorry i'll fix